--- reaxc_valence_angles.cpp.orig	2019-08-07 00:01:57.000000000 +0200
+++ reaxc_valence_angles.cpp	2019-10-25 11:33:33.431570135 +0200
@@ -257,9 +257,9 @@
                 expval7 = exp( -p_val7 * workspace->Delta_boc[j] );
                 trm8 = 1.0 + expval6 + expval7;
                 f8_Dj = p_val5 - ( (p_val5 - 1.0) * (2.0 + expval6) / trm8 );
-                Cf8j = ( (1.0 - p_val5) / SQR(trm8) ) *
-                  ( p_val6 * expval6 * trm8 -
-                    (2.0 + expval6) * ( p_val6*expval6 - p_val7*expval7 ) );
+                Cf8j = (1.0 - p_val5) * ( p_val6 * expval6 / trm8 -
+                  ((2.0 + expval6) / trm8) * (( p_val6*expval6 -
+                    p_val7*expval7 ) / trm8) );
 
                 theta_0 = 180.0 - theta_00 * (1.0 -
                                               exp(-p_val10 * (2.0 - SBO2)));
